Makefile

```diff
@@ -2,7 +2,7 @@
 
 DOCKER_NAME_RUST_OPTIMIZER := "cosmwasm/rust-optimizer"
 DOCKER_NAME_WORKSPACE_OPTIMIZER := "cosmwasm/workspace-optimizer"
-DOCKER_TAG := 0.12.2
+DOCKER_TAG := 0.12.2_rust_port
```

Comment:
Let's just use `0.12.3` here for consistency. `0.12.2` is still unreleased, but still.

Reply:
Sure. This was just a temporary version number for demonstration :)